scriptmodules/supplementary/runcommand/runcommand.sh

```diff
@@ -412,6 +412,9 @@ function main_menu() {
     local cmd
     local choice
 
+    local user_menu=0
+    [[ -d "$CONFIGDIR/all/runcommand-menu" && -n "$(ls -A "$CONFIGDIR/all/runcommand-menu")" ]] && user_menu=1
```

Comment:
suggestion to avoid empty/non-sh files:
```
[[ -n "$(find "$CONFIGDIR/all/runcommand-menu" -type f -name "*.sh" ! -size 0 2>/dev/null)" ]] && user_menu=1
```